pfhedge/nn/modules/hedger.py

```diff
@@ -207,7 +207,7 @@ def forward(self, input: Tensor) -> Tensor:
     def extra_repr(self) -> str:
         return "inputs=" + str(self.inputs)
 
-    def get_input(self, time_step: Optional[int]) -> Tensor:
+    def get_input(self, time_step: Optional[int], derivative: Derivative) -> Tensor:
```

Comment:
nit: Maybe you have notice this, but this order might be better as derivative is required while time_step is optional (I suggested in a wrong way in the previous issue comment, sorry about this)

```suggestion
    def get_input(self, derivative: Derivative, time_step: Optional[int]) -> Tensor:
```